site/gatsby-site/src/elements/ToggleSwitch/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
Is this something that flow bite -react doesn't provide?

Reply:
At the moment that I code this feature the [ToggleSwitch flowbite component](https://flowbite.com/docs/components/forms/#toggle-switch) didn't work because it uses `useId` React 18 hook. That's why I decided to move forward and implement our own ToogleSwitch element.
I can try now with the `react-dom` 18 update and see if it works now.

Reply:
Well, it looks like after the `react-dom` upgrade to the version `18` the [ToggleSwitch flowbite component](https://flowbite-react.com/forms) works as expected 👌